docker-compose.dev.yml

```diff
@@ -0,0 +1,59 @@
+# Run options:
+# 1. There are no options - This add additional tools to aid in the development of Libre Photos
+#   run cmd: docker-compose up -f docker-compose.yml -f docker-compose.dev.yml -d
+# 2. Current added tools:
+#    pgadmin User admin@admin pass admin port 3001
+
+# DO NOT EDIT
+# The .env file has everything you need to edit.
+# Run options:
+# 1. Use prebuilt images (preferred method):
+#   run cmd: docker-compose up -d
+# 2. Build images on your own machine:
+#   build cmd: COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
+#   run cmd: docker-compose up -d
+
+version: '3.8'
+services:
+
+  frontend:
+    tty: true
+    environment:
+    - DEBUG=1
+    build:
+      context: .
+      dockerfile: frontend/Dockerfile
+    volumes:
+      - ${codedir}/frontend:/usr/src/app
+
+  backend:
+    tty: true
+    stdin_open: true
+    environment:
+      - DEBUG=1
+    build:
+      context: .
+      dockerfile: backend/Dockerfile
+      args:
+        DEBUG: 1
+    volumes:
+      - ./backend/entrypoint.sh:/entrypoint.sh
+      - ${myPhotos}:/data
+      - ${proMedia}:/protected_media
+      - ${logLocation}:/logs
+      - ${codedir}/backend:/code
```

Comment:
This should be the name of the repository "librephotos"

Reply:
I don't get what do you mean with "this".

These are the mounted directories selected in .env file

Reply:
Ok reading the comment below I'm trying to guess you mean that backend should be replaced by  librephotos right? no problem for, me.

Reply:
Yes, that's what I want :+1: